extensions/adobe/experience/primetime-authentication/authorization.schema.json

```diff
@@ -0,0 +1,113 @@
+  "definitions": {
+    "rational": {
+      "$id": "https://ns.adobe.com/experience/primetime-authentication/authorization/resource",
+      "title": "Resource",
+      "description":
+        "A representation of the media resource the end user wants to access.",
+      "properties": {
+        "xdm:channel": {
+          "title": "Resource channel",
+          "description": "The resource channel",
+          "type": "string"
+        },
+        "xdm:raw": {
+          "title": "Raw resource",
+          "description": "The raw representation of the resource",
+          "type": "string"
+        },
+        "xdm:authorized" : {
+          "title" : "Authorization status",
+          "description": "Authorization decision for each resource",
+          "type": "boolean"
+        }
+      },
+      "required": ["xdm:channel", "xdm:authorized"]
+    },
+    "authorization": {
+      "properties": {
+        "xdm:idp": {
+          "title": "MVPD Identifier",
+          "type": "string",
+          "description":
+            "The MVPD with whom the end user is authorized or tries to authorize"
+        },
+        "xdm:technologyProvider": {
+          "title": "Technology Provider",
+          "type": "string",
+          "description":
+            "The partner which provides the infrastructure for IdP authorization endpoints"
+        },
+        "xdm:broker": {
+          "title": "Authorization Broker",
+          "type": "string",
+          "description":
+            "External authorization provider which might intermediate the authorization flow"
+        },
+        "xdm:authorizationType": {
+          "title" : "Authorization Type",
+          "type": "string",
+          "description": "The authorization type",
+          "meta:enum": {
+            "authoritative" : "The flow through which the end user obtains an authoritative authorization decision.",
+            "non-authoritative" : "The flow through which the end user obtains a non-authoritative authorization decision useful initial content filtering"
+          }
+        },
+        "xdm:resources" : {
+          "title": "Resources",
+          "type": "array",
+          "description": "The list resources for which the user request authorization",
+          "$ref": "#/definitions/resource"
+        },
+        "xdm:timeToLive": {
+          "title": "Time To Live",
+          "type": "integer",
+          "description":
+            "Authorization token time to live in seconds."
+        },
+        "xdm:latency": {
+          "title": "Request latency",
+          "type": "integer",
+          "description":
+            "The time the authorization network calls takes to return back to adobe pass services in milliseconds."
+        },
+        "xdm:failReason": {
+          "title" : "Fail Reason",
+          "type": "string",
+          "description": "The reason for failed authentication",
```

Comment:
I believe it should state `The reason for failed authorization `

Reply:
Done.